File: src/csa/week0/MatrixFormat.java
package csa.week0;
import csa.util.Menu;
import csa.util.Option;

public class MatrixFormat extends Option {

    public MatrixFormat() {
        optionName = "Matrix Format";
    }

    public void tester() {
        Option one = new Numpad();
        Option two = new Keyboard();

        Option[] options = new Option[] {one, two};

        Menu menu = new Menu();
        menu.run("\nNumpad or Keyboard?", options);
    }
}

File: src_py/week0/matrix_format.py
from src_py.util.option import Option


class MatrixFormat(Option):
    def __init__(self):
        super().__init__()
        self.name = "Matrix Format"
        self.numpad = [[1, 2, 3],
                       [4, 5, 6],
                       [7, 8, 9],
                       [" ", 0, " "]]

    def __call__(self):
        for i in self.numpad:
            for j in i:
                print(j, "", end="")
            print()
